servers/physics_3d/body_3d_sw.cpp

```diff
@@ -102,6 +105,11 @@ void Body3DSW::update_inertias() {
 				inertia_tensor += shape_inertia_tensor + (Basis() * shape_origin.dot(shape_origin) - shape_origin.outer(shape_origin)) * mass;
 			}
 
+			// Set the inertia to a valid value when there are no valid shapes
```

Comment:
The comment should end with a period to follow the style guide.
(lots of comments don't respect this style guide in the physics servers but it's better to do it on new code anyway)

Reply:
Got it. I'll change a few of the other comments in the file while I'm at it.